lib/sitemap/sitemap_generator.rb

```diff
@@ -13,10 +14,10 @@ def self.sitemap_limit
   def get_all_documents
     Enumerator.new do |yielder|
       # Hard-code the site root, as it isn't listed in any search index
-      yielder << "/"
+      yielder << homepage_document
 
       @sitemap_indices.each do |index|
-        index.all_document_links(EXCLUDED_FORMATS).each do |document|
+        index.all_documents(exclude_formats: EXCLUDED_FORMATS).each do |document|
           if document
```

Comment:
Good question. In principle this is possible because it's not a required field. In practice, every document in ES does have a `link` field. (I've just checked on production.)

Do you think it's worth checking during sitemap generation?